ivy/src/main/scala/sbt/CustomPomParser.scala

```diff
@@ -95,10 +108,10 @@ object CustomPomParser {
       val mergeDuplicates = IvySbt.hasDuplicateDependencies(md.getDependencies)
 
       val unqualify = toUnqualify(filtered)
-      if (unqualify.isEmpty && extraDepAttributes.isEmpty && !convertArtifacts && !mergeDuplicates)
```

Comment:
not in this PR.  mergeDuplicates has some work it still does which is useful, we just need to figure otu either an alternative mechanism *or* make that mechanism work.